wandb/sdk/wandb_sweep.py

```diff
@@ -98,7 +98,7 @@ def my_train_func():
         env.set_project(project)
 
     # Make sure we are logged in
-    wandb_login._login(_silent=True)
+    wandb_login._login(_silent=True, _disable_warning=True)
```

Comment:
Maybe instead of silencing it, check if it's necessary to try logging in in the first place? @dannygoldstein?
```suggestion
    if wandb.run is not None:
        wandb_login._login(_silent=True)
```

Reply:
It's just about where to do the check -- `_login` does exactly that down the line.